components/Modal/CreateImageUpload.js

```diff
@@ -942,18 +1090,41 @@ class CreateImageUploadModal extends React.Component {
             </div>
             <div className="pf-c-form__group">
               <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
-                <label htmlFor="container-input" className="pf-l-flex__item">
-                  <span className="pf-c-form__label-text">Storage container</span>
+                <label htmlFor="storage-container-input" className="pf-l-flex__item">
+                  <span className="pf-c-form__label-text">
+                    <FormattedMessage defaultMessage="Storage container" />
+                  </span>
                   <span className="pf-c-form__label-required" aria-hidden="true">
                     &#42;
                   </span>
                 </label>
+                <Popover
+                  id="bucket-popover"
+                  bodyContent={
+                    <TextContent>
+                      <FormattedMessage defaultMessage="Provide the Blob container to which the image file will be uploaded. You can find containers under the" />
+                      <strong>
+                        <FormattedMessage defaultMessage=" Blob service " />
+                      </strong>
+                      <FormattedMessage defaultMessage="section of a storage account. You can find storage accounts on the" />
+                      <strong>
+                        <FormattedMessage defaultMessage=" Storage accounts " />
+                      </strong>
+                      <FormattedMessage defaultMessage="page in the {azure} portal." values={{ azure: "Azure" }} />
+                    </TextContent>
```

Comment:
When I review the .pot file for this string, the strings are broken up. For example, one of the strings is presented to translators like this:
```
#: build/messages/components/Modal/CreateImageUpload.json
#. [components.Modal.2862073206]
#. defaultMessage is:
#. Provide the Blob container to which the image file will be uploaded. You can find containers under the
msgid "Provide the Blob container to which the image file will be uploaded. You can find containers under the"
msgstr ""
```

When strings are provided to translators, they should be complete phrases or sentences. The best solution I can think of for handling this is how it's handled [here](https://github.com/osbuild/cockpit-composer/blob/master/components/ListView/SourcesListItem.js#L52).

There are a couple of other strings in this PR that require similar fixes.